WIP: Fixup the documentation on extension bundling (#5710)
jupyterlab/jupyterlab: docs/source/developer/extension_dev.rst

package.

The following file types are also supported (both in JavaScript and
CSS): ``json``, ``html``, ``jpg``, ``png``, ``gif``, ``svg``,

> ``js.map``, ``woff2``, ``ttf``, eot.

Reviewer: ```suggestion
``js.map``, ``woff2``, ``ttf``, ``eot``.
```

If your package uses any other file type it must be converted to one of
the above types or `include a loader in the import statement <https://webpack.js.org/concepts/loaders/#inline>`__.
If you include a loader, the loader must be importable at build time, so if
it is not already installed by JupyterLab, you must add it as a dependency
of your extension.

If your JavaScript is written in any other dialect than

> EMCAScript 5 it must be converted using an appropriate tool.

Author: ```suggestion
EMCAScript 6 (2015) it should be converted using an appropriate tool.
```

You can use Webpack to pre-build your extension to use any of it's features

> not enabled in our build config. To build a compatible package sett

Reviewer: ```suggestion
not enabled in our build config. To build a compatible package set
```
